README.Rmd

```diff
@@ -131,11 +131,12 @@ Each `xportr_` function has been written in a way to take in a part of the speci
 
 ```{r, warning = FALSE, message=FALSE, eval=TRUE}
 adsl %>%
-  xportr_type(var_spec, "ADSL", verbose = "warn") %>%
-  xportr_length(var_spec, "ADSL", verbose = "warn") %>%
-  xportr_label(var_spec, "ADSL", verbose = "warn") %>%
-  xportr_order(var_spec, "ADSL", verbose = "warn") %>%
-  xportr_format(var_spec, "ADSL") %>%
+  xportr_domain_name("ADSL") %>%
```

Comment:
Is this still needed?

Reply:
What do you suggest?

This would look cleaner :sunglasses: even cleaner after #199

```R
  xportr_metadata(var_spec, "ADSL") %>%
  xportr_type(verbose = "warn") %>%
  xportr_length(verbose = "warn") %>%
  xportr_label(verbose = "warn") %>%
  xportr_order(verbose = "warn") %>%
  xportr_format() %>%
  xportr_df_label(dataset_spec, "ADSL") %>%
  xportr_write("adsl.xpt")
```